tests/cypress/integration/components/a-z-nav.js

```diff
@@ -0,0 +1,46 @@
+describe('setting screen res', function () {
```

Comment:
This should describe the whole test such as "Navigation A-Z tests" rather than setting the screen resolution, as this describes the whole tests not just the first action.

Reply:
updated the description